src/main/java/com/ge/predix/log4j1/PredixLayoutPattern.java

```diff
@@ -54,4 +53,22 @@ protected String convert(final LoggingEvent event) {
                     + event.getMessage();
         }
     }
+    
+    private List<List<String>> getStackTrace(final LoggingEvent event) {
+        List<List<String>> exceptions = new ArrayList<>();
+        Throwable throwable = event.getThrowableInformation().getThrowable();
+        while (throwable != null) {
+            List<String> stack = new ArrayList<>();
+            stack.add(throwable.getClass().getName()
+                    + (!StringUtils.isEmpty(throwable.getMessage()) ? ": " + throwable.getMessage() : ""));
+            for (StackTraceElement element : throwable.getStackTrace()) {
+                stack.add("at " + element.toString());
```

Comment:
The json structure with 4 spaces indent is just as confusing as without indenting, maybe not?